org.feichtmeier.Musicpod.json

```diff
@@ -0,0 +1,228 @@
+                                        "type": "git",
+                                        "tag-pattern": "^n([\\d.]+)$"
+                                   }
+                              }
+                         ]
+                    },
+                    {
+                         "name": "libass",
+                         "cleanup": [
+                              "/include",
+                              "/lib/*.la",
+                              "/lib/pkgconfig"
+                         ],
+                         "config-opts": [
+                              "--disable-static"
+                         ],
+                         "sources": [
+                              {
+                                   "type": "archive",
+                                   "url": "https://github.com/libass/libass/releases/download/0.17.1/libass-0.17.1.tar.xz",
+                                   "sha256": "f0da0bbfba476c16ae3e1cfd862256d30915911f7abaa1b16ce62ee653192784",
+                                   "x-checker-data": {
+                                        "type": "anitya",
+                                        "project-id": 1560,
+                                        "stable-only": true,
+                                        "url-template": "https://github.com/libass/libass/releases/download/$version/libass-$version.tar.xz"
+                                   }
+                              }
+                         ]
+                    },
+                    {
+                         "name": "uchardet",
+                         "buildsystem": "cmake-ninja",
+                         "config-opts": [
+                              "-DCMAKE_BUILD_TYPE=Release",
+                              "-DBUILD_STATIC=0"
+                         ],
+                         "cleanup": [
+                              "/bin",
+                              "/include",
+                              "/lib/pkgconfig",
+                              "/share/man"
+                         ],
+                         "sources": [
+                              {
+                                   "type": "archive",
+                                   "url": "https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz",
+                                   "sha256": "e97a60cfc00a1c147a674b097bb1422abd9fa78a2d9ce3f3fdcc2e78a34ac5f0",
+                                   "x-checker-data": {
+                                        "type": "anitya",
+                                        "project-id": 9265,
+                                        "stable-only": true,
+                                        "url-template": "https://www.freedesktop.org/software/uchardet/releases/uchardet-$version.tar.xz"
+                                   }
+                              }
+                         ]
+                    },
+                    {
+                         "name": "squashfs-tools",
```

Comment:
This should be cleaned up, because you are not using it at runtime, only to extract things at build time.

`"cleanup": ["*"]`

Reply:
yes